server/src/main/java/org/candlepin/controller/CandlepinPoolManager.java

```diff
@@ -697,6 +695,41 @@ protected int cleanupExpiredPoolsImpl() {
         return entitlementsToRegen;
     }
 
+    Set<Pool> revokeEntitlementsFromOverflowingPools(List<Pool> pools) {
```

Comment:
Is this default visibility for a reason?  Not a big deal, but I don't normally see methods that we keep package-protected.